src/components/UiLibrary.js

```diff
@@ -1,32 +1,39 @@
+import AddObsButton from "components/SharedComponents/Buttons/AddObsButton";
+import Button from "components/SharedComponents/Buttons/Button";
+import EvidenceButton from "components/SharedComponents/Buttons/EvidenceButton";
+import SecondaryCTAButton from "components/SharedComponents/Buttons/SecondaryCTAButton";
+import Body1 from "components/SharedComponents/Typography/Body1";
+import Body2 from "components/SharedComponents/Typography/Body2";
+import Body3 from "components/SharedComponents/Typography/Body3";
+import Body4 from "components/SharedComponents/Typography/Body4";
+import Heading1 from "components/SharedComponents/Typography/Heading1";
+import Heading2 from "components/SharedComponents/Typography/Heading2";
+import Heading3 from "components/SharedComponents/Typography/Heading3";
+import Heading4 from "components/SharedComponents/Typography/Heading4";
+import List1 from "components/SharedComponents/Typography/List1";
+import List2 from "components/SharedComponents/Typography/List2";
+import Subheading1 from "components/SharedComponents/Typography/Subheading1";
+import ViewWithFooter from "components/SharedComponents/ViewWithFooter";
 import {
   ScrollView,
-  Text,
   View
 } from "components/styledComponents";
 import React from "react";
 import { Alert } from "react-native";
 
-import AddObsButton from "./SharedComponents/Buttons/AddObsButton";
-import Button from "./SharedComponents/Buttons/Button";
-import EvidenceButton from "./SharedComponents/Buttons/EvidenceButton";
-import SecondaryCTAButton from "./SharedComponents/Buttons/SecondaryCTAButton";
-import ViewWithFooter from "./SharedComponents/ViewWithFooter";
-
 /* eslint-disable i18next/no-literal-string */
 /* eslint-disable react/no-unescaped-entities */
 const UiLibrary = ( ) => (
   <ViewWithFooter>
     <ScrollView className="px-5">
       {/* TODO replace these text components with our typography header components */}
-      <Text>
+      <Body1>
         All the re-usable UI components we've got. If you're making a new UI
         component, please put it here first and try to show what it looks
         like with different property configurations.
-      </Text>
-
-      <Text className="text-xl">Buttons</Text>
-
-      <Text className="text-lg">Button</Text>
+      </Body1>
+      <Heading1>Buttons</Heading1>
+      <Subheading1>Button</Subheading1>
```

Comment:
I don't *think* the intent of `Subheading1` is to show hierarchy, so let's use Heading2 here and everywhere else you've used `Subheading1`.